client/landing/stepper/declarative-flow/internals/steps-repository/playground/components/playground-iframe/index.tsx

```diff
@@ -46,7 +46,13 @@ export function PlaygroundIframe( {
 					setPlaygroundError( 'UNKNOWN_ERROR' );
 				}
 			} );
-	}, [ playgroundError ] );
+	}, [
+		playgroundClient,
+		playgroundError,
+		recommendedPHPVersion,
+		setPlaygroundClient,
+		setSearchParams,
+	] );
```

Comment:
@zaerl FYI, I had to undo this change. It was deliberate to keep only `playgroundError` as the only dependency capable of causing a re-render, otherwise PG breaks. I have added a comment for eslint.

Reply:
It's ok, thanks.